tests.py

```diff
@@ -1,3 +1,4 @@
+#! /usr/bin/env python3
```

Comment:
Shouldn't this be `#! /usr/bin/env python` instead of assuming python 3 availability and preference?